src/pybind.cpp

```diff
@@ -9,6 +9,8 @@
 
 namespace py = pybind11;
 extern int register_server(unsigned long loop_ptr, server_config_t config);
+extern int set_log_level(std::string level);
```

Comment:
i suggest move register_server set_log_level.. to header file

Reply:
how about common.h? we can also move config.h to new header file

Reply:
no， just include log.h in pybind.cpp

Reply:
updated.